settings_dialog.py

```diff
@@ -189,6 +208,10 @@ def accept(self) -> None:
         config.exported_tag = self.tag_edit.text()
         config.hidden_fields = self.hidden_fields.values()
         config.timeout_seconds = self.web_timeout_spinbox.value()
+        config.sentence_min_length = self.sentence_min_length.value()
+        config.sentence_max_length = self.sentence_max_length.value() \
+            if self.sentence_max_length.value() > self.sentence_min_length.value()\
+                 or self.sentence_max_length.value() == 0 else self.sentence_min_length.value() +1
```

Comment:
Is this complex assignment justified? If the max value is less than the min value, the api will just return no results, right?